src/LessonTestT/LessonTestTask1.java

```diff
@@ -0,0 +1,8 @@
+package LessonTestT;
+
+public class LessonTestTask1 {
+    public static void main(String[] args) {
+        System.out.println("Грачи прилетели");
+    }
+
+}
```

Comment:
@EgorTor11 Не оставляй бессмысленно пустые строки
```suggestion
    }
}
```